### PR TITLE
ADO 128475: Missing ALW for clients

### PR DESCRIPTION
**Pull request by alex-solo**
### Description

- When single income is more advantageous, we should use that to calculate ALW 

#### List of proposed changes:

- more details and scenarios to test against are listed in the ADO task
-

### What to test for/How to test

Any additional edge cases you can think of, please go ahead and test

### CODE REVIEW

File: utils/api/benefitHandler.ts [hunk 1]
```diff
@@ -891,6 +891,38 @@ export class BenefitHandler {
                     .calculatedBasedOnIndividualIncome
                 )
 
+              // If client is eligible for ALW, need to recalculate estimate based on individual income
+              if (clientAlw.eligibility.result === 'eligible') {
+                const tempClientAlw = new AlwBenefit(
```
Comment (MarcoGoC) [inline utils/api/benefitHandler.ts]: not sure is necessary, because the correct amount is on applicantAlwCalcSingle

**Review by MarcoGoC — APPROVED** [review]
fixes the main 'bug' as requested
